backend/src/main/resources/mapper/SqlRecordMapper.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
+<mapper namespace="com.hufudb.onedb.backend.mapper.SqlRecordMapper">
+
+    <resultMap id="BaseResultMap" type="com.hufudb.onedb.backend.entity.SqlRecord">

```

Comment:
You need to check if it is necessary to manually assign the resultMap when the column name is the same with the name in java.